### PR TITLE
NEXUS-21778 [Helm Bugs] List of maintainers is not available

### PR DESCRIPTION
**Pull request by proteus1121**
https://issues.sonatype.org/browse/NEXUS-21778

### CODE REVIEW

File: nexus-repository-helm/src/main/java/org/sonatype/repository/helm/HelmAttributes.java [hunk 1]
```diff
@@ -36,11 +37,31 @@ public HelmAttributes(final Map<String, Object> attributesMap) {
...
+  private void putInMap(final HelmProperties helmProperties, final Object value) {
+    switch (helmProperties){
+      case MAINTAINERS:
+        // convert map: name: Bitnami
+        //              email: containers@bitnami.com
+        //              name: wbuchwalter
+        //              email: wibuch@microsoft.com
+        // in list: Bitnami : containers@bitnami.com , wbuchwalter : wibuch@microsoft.com
+        List<Map<String, String>> maintainersMap = (List<Map<String, String>>) value;
+        List<String> maintainers = maintainersMap.stream()
```
Comment (chernikovmaksym) [inline nexus-repository-helm/src/main/java/org/sonatype/repository/helm/HelmAttributes.java]: Question: By default any attributes are stored as nested key-value map. Can storing attributes this way break anything? Like search, for example?

Comment (chernikovmaksym) [reply nexus-repository-helm/src/main/java/org/sonatype/repository/helm/HelmAttributes.java]: I think it's better to ask someone from community about such cases

File: nexus-repository-helm/src/main/java/org/sonatype/repository/helm/HelmAttributes.java [hunk 2]
```diff
@@ -36,11 +37,31 @@ public HelmAttributes(final Map<String, Object> attributesMap) {
...
       Optional<HelmProperties> propertyOpt = HelmProperties.findByPropertyName(key);
       if (value != null && propertyOpt.isPresent()) {
-        attributesEnumMap.put(propertyOpt.get(), value);
+        putInMap(propertyOpt.get(), value);
       }
     });
   }
 
+  private void putInMap(final HelmProperties helmProperties, final Object value) {
+    switch (helmProperties){
```
[the remark this answers is not in the document]
Comment (proteus1121) [reply nexus-repository-helm/src/main/java/org/sonatype/repository/helm/HelmAttributes.java]: Done

File: nexus-repository-helm/src/main/java/org/sonatype/repository/helm/HelmAttributes.java (hunk 1 above)
Comment (proteus1121) [reply nexus-repository-helm/src/main/java/org/sonatype/repository/helm/HelmAttributes.java]: I think that it depends on search feature implementation, I fixed just representing of this attribute. Anyway as I can see we cant save in NastedAttribute any Map (check NestedAttributesMap.set() implementation)

**Pull request merged**